src/script/auth/page/SingleSignOnForm.tsx

```diff
@@ -187,11 +187,11 @@ class _SingleSignOnForm extends React.PureComponent<Props & ConnectedProps & Dis
           }
           default: {
             this.setState({ssoError: error});
-            const isValidationError = Object.values(ValidationError.ERROR).some(errorType =>
-              error.label.endsWith(errorType)
+            const isValidationError = Object.values(ValidationError.ERROR).some(
+              errorType => error.label && error.label.endsWith(errorType)
             );
             if (!isValidationError) {
-              throw error;
+              console.warn('SSO authentication error', JSON.stringify(Object.entries(error)), error);
```

Comment:
would it make sense to throw the error as before, but log the error unconditionally?

```
             console.info('SSO authentication error', JSON.stringify(Object.entries(error)), error);
             if (!isValidationError) {
                 throw error;
             }
```

no strong opinion there, i'd be happy with your change as is (given it works as discussed :).  and i haven't tested the `console.info` thing.

Reply:
Basically we filter for known errors not related to SSO. In other words all SSO errors are logged here.